### PR TITLE
Add config generator and light tests

### PR DESCRIPTION
**Pull request by andrewmcodes**
This all works but the tests are really hacky - couldn't find any good way to stub the file creation in Minitest. If everything except the generator was asked to be removed I wouldn't be upset lol

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -2,13 +2,6 @@
...
-# You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
-
-# (If you use this, don't forget to add pry to your Gemfile!)
-# require "pry"
-# Pry.start
-
-require "irb"
-IRB.start(__FILE__)
+Pry.start
```
Comment (afomera) [inline bin/console:7]: You don't like binding.irb? It's super handy haha.

Comment (andrewmcodes) [reply bin/console:7]: Never used binding.irb in my life but this is a preference and can easily be reverted.

File: test/applocale/ruby_test.rb [hunk 2]
```diff
@@ -1,11 +1,7 @@
 require "test_helper"
 
-class Applocale::RubyTest < Minitest::Test
```
Comment (afomera) [inline test/applocale/ruby_test.rb:3]: file is named ruby_test, but we've updated the class name to be VersionTest. Mind renaming the file to version_test.rb?

File: bin/console (hunk 1 above)
Comment (afomera) [reply bin/console:7]: I'm okay leaving pry in there as its pretty common.

File: test/applocale/ruby_test.rb (hunk 2 above)
Comment (andrewmcodes) [reply test/applocale/ruby_test.rb:3]: good catch! Fixed.

**Pull request merged**